Sources/XcodesKit/RuntimeInstaller.swift

```diff
@@ -38,18 +38,16 @@ public class RuntimeInstaller {
             }
         }
 
-
-
-
+        let unusableCount = installed.removeAll { $0.state == "Unusable" }.count
         installed.forEach { runtime in
             let resolvedBetaNumber = downloadablesResponse.sdkToSeedMappings.first {
-                $0.buildUpdate == runtime.build
+                $0.buildUpdate == runtime.build!
```

Comment:
Can we refactor this section a bit? 

Can we **not** use force unwrap. As a rule, I don't use them. Totally agree that it should _never_ happen but it's bitten me in the past. Perhaps making an overloaded `PrintableRuntime()` to allow optionals?

Reply:
I'm not sure how best to achieve this, feel free to make any changes.

Reply:
Just my 2 cents here, personally I also try to avoid them if I can, but in this case, I would prefer a force unwrap to fail than hiding the error and providing an empty string as a default value, in that case, we would know that `xcrun simctl runtime list` printed something that we didn't account for, and there is probably a new state of a runtime that we should write code for. I see it as a `assert`/`precondition` and not a code smell.

Reply:
@MattKiazyk can you please give an update on how you would like to proceed here?